pool/store/mongodb_store.py

```diff
@@ -175,5 +175,7 @@ async def get_recent_partials(self, launcher_id: bytes32, count: int) -> List[Tu
         for item in results:
             ret.append((uint64(item['timestamp']), uint64(item['difficulty'])))
         
-        self.log.info(f"{launcher_id}, current difficulty : {ret[0][1]}, partials count : {len(ret)}")        
+        if(len(ret) > 0) :
```

Comment:
we dont really need () 
if len(ret) > 0:

Reply:
If we don't have if check, farmer joining has no partial will failed and crash the code. Result is farmer will not able to join our pool

Reply:
I mean does not need outside parathesis.

Reply:
we also need to make sure ret[0][1] available